src/products/views/ProductUpdate/handlers/index.ts

```diff
@@ -133,7 +132,7 @@ export function createUpdateHandler(
           title: data.seoTitle,
         },
         slug: data.slug,
-        taxCode: data.changeTaxCode ? data.taxCode : null,
+        taxClass: data.taxClassId,
```

Comment:
```suggestion
        taxClassId: data.taxClassId,
```

Reply:
Actually I can't change this. This is how product input is defined in the API